ehrapy/anndata/anndata_ext.py

```diff
@@ -482,8 +482,8 @@ def _update_uns(
     2.) Moving `col1` from `obs` to `X`: `col1` is either numerical or non_numerical, so add it to the corresponding entry in `uns`
     """
     moved_columns_set = set(moved_columns)
-    num_set = set(adata.uns["numerical_columns"])
-    non_num_set = set(adata.uns["non_numerical_columns"])
+    num_set = set(adata.uns["numerical_columns"].copy())
```

Comment:
Later down we are copying the whole `.uns`. Are both copies required?

Reply:
Yes, because were mutating the dicts here (updating them) so we need those copies. Otherwise it would mutate the original lists/sets as well.